client/components/input-chrono/docs/example.jsx

```diff
@@ -17,40 +17,44 @@ import Card from 'components/card';
 /**
  * Date Picker Demo
  */
-export default localize(React.createClass({
-	displayName: 'InputChrono',
-
-	mixins: [ PureRenderMixin ],
-
-	getInitialState() {
-		return {
-			date: this.props.moment(),
-		};
-	},
-
-	componentWillMount() {
-		var self = this;
-		this.interval = setInterval( function() {
-			var date = self.moment( self.state.date );
-			date.hours( date.hours() + 1 );
-			self.setState( { date: date } );
-		}, 1000 );
-	},
-
-	componentWillUnmount() {
-		clearInterval( this.interval );
-	},
-
-	onSet( date ) {
-		console.log( `date: %s`, date.toDate() );
-		this.setState( { date: date } );
-	},
-
-	render() {
-		return (
-			<Card style={ { width: '300px', margin: 0 } }>
-				<InputChrono value={ this.state.date.calendar() } onSet={ this.onSet } />
-			</Card>
-		);
-	},
-}));
+const InputChronoExample = localize(
```

Comment:
The codemod I have up for review should fix these cases properly, allowing us to export the base component for examples. Unless I'm wrong, the ideal format (ignoring `.createClass` vs `extends...`) here would be something like:

```js
export const InputChronoExample = React.createClass(...);

export default enhance( InputChronoExample );
```

Using `import { InputChronoExample } from ...` for the example.

Reply:
Ignore the above - I actually missed the point that this is the example itself!

Maybe it's best instead to use a pattern like so:

```jsx
render() {
  <div>
    <InputChrono
      value={ this.state.date.calendar() }
      onSet={ this.onSet }
      translate={ noop }
    />
  </div>
}
```

I've seen this elsewhere and it removes some of the extra potentially confusing noise of setting the `displayName`, though potentially adds some confusion around the `translate=noop`... A comment there could help ease that confusion though.

Reply:
FWIW, I use the `translate={ noop }` (well, `identity`) pattern a lot in tests. In this case, it's a bit different, since we aren't really feeding it to the component, but need it inside the example; and it's not just `translate` but `moment`, which we can't (and don't want to) easily stub with `identity`.

Reply:
> (well, identity)

That's what I meant 😄 

Good example with moment - not something I considered!
In that case, I think it's fair to force the `displayName` - in examples that is :)